### ISSUE TITLE
Command-Shift-P fuzzy command menu

### ISSUE THREAD

**Issue by Spacerat** [pebble/cloudpebble#176]
In addition to the Cmd-P menu already built for file searching, the fuzz-search prompt supports executing commands. The functionality is present already but disabled for now until we decide on a reasonable set of commands to support from the get-go, name them properly, and fully test/implement them all.

List of commands built so far:
* Show Phone Logs
* Show Emulator Logs
* Show Last Build Log
* Clear App Logs
* Run (is the same as pressing the green button)
* Compilation (takes you to the 'Compilation' window)
* Settings (takes you to the settings window)

Things we could add (in order, top = most important in my opinion)
* Navigation to Timeline and Github windows
* Build
* New File, New Resource
* Git commands (push, pull, etc)
* File and resource operations: Save, Reload, Rename, Delete
* Install and run (for phone, and each emulator)
* Toggle Fullscreen
* Take screenshot
* Insert/Delete pin
* Download project as zip

If anyone's got any opinions on inclusion/exclusion or additional ideas for commands, feel free to add them!

![fuzzy-command screenshot](http://i.imgur.com/Kac60E7.png)

**Comment by thunsaker** [comment]
* Re-indent/Format Selection or File
* Reload Auto-Complete
* Toggle JSHint (Check Javascript)
* Documentation Search

**Comment by Spacerat** [comment]
I've enabled this feature in the [CloudPebble Beta](https://beta.cloudpebble.net/ide/). I've not included that many more commands than the ones I listed, but I'll continue to add more over time.

That being said, it's relatively easy to add global commands (i.e. ones which don't require any context). Here's some examples: https://github.com/pebble/cloudpebble/blob/116352128f88740a83f9bd43b80a9b8e163016ee/ide/static/ide/js/compile.js#L162
so if you've got any commands you're burning to add in ASAP, feel free to submit a PR for them!
